app/src/userpages/modules/permission/actions.js

```diff
@@ -113,15 +113,17 @@ const saveRemovedResourcePermissionFailure = (resourceType: ResourceType, resour
     permission,
 })
 
-export const getResourcePermissions = (resourceType: ResourceType, resourceId: ResourceId) => async (dispatch: Function) => {
+export const getResourcePermissions = (resourceType: ResourceType, resourceId: ResourceId, notify: boolean = true) => async (dispatch: Function) => {
```

Comment:
`getResourcePermissions` gives a 403, which is weird. I don't see a scenario where the permissions endpoint should respond with a 403 for a logged in users, really.

Reply:
e.g. if you're logged in as A and trying to access B's resources, but you don't have permission?

Reply:
Not really trying to access. Just checking the permissions gives a 403. I'd imagine that checking my own permissions to a resource should simply be a `200` followed by the list of things I can do to a resource.

Reply:
@timoxley, the problem here is that I shared a canvas with my other account. Asking for permissions to that canvas from the other account ends up as a 403.

<img width="682" alt="Screenshot 2019-11-04 12 35 21" src="https://user-images.githubusercontent.com/320066/68118792-23ab7480-ff01-11e9-9d6a-90f2658df107.png">

I may not understand the permissions system fully.